[FIX] 0043730: Failed test: MainControls - System Info: show

chfsx opened the pull request:
This should fix the issue described in https://mantis.ilias.de/view.php?id=43730
(I changed the order of the methods since my IDE has issues with the order, IDK if this is relevant).

klees commented:
Hi @chfsx,

we (as in @Amstutz and me) have some trouble seeing the actual change. Could you turn this into 2 commits, one for the formatting, one for the bugfix?

Thanks a lot!

chfsx committed:
[FIX] 0043730: Failed test: MainControls - System Info: show

chfsx commented:
Please note, the commit has mistakenly been merged by me, I reverted this now in release_10.

I also updated this PR to make more clear what changes. I therefore has to introduce JS which doesn't comply with all guidelines. I hope it doesn't get any more complicated now, that would be disproportionate to the extent of what has been changed.

In short:
The problem is, that resizing the window triggered the "toggleClass" hundreds of times which resulted in the issue described in https://mantis.ilias.de/view.php?id=43730

klees commented:
Thx, @chfsx!

klees commented:
Picked to trunk.
